dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -577,6 +584,24 @@ public static <T> T getArbitraryURL(String url, GenericType<T> type, ApiClient c
         return getArbitraryURL(url, type, client, "application/zip");
     }
 
+    /**
+     * This retrieves a resource file using getResourceAsStream, which works for retrieving resource files packaged in a jar.
+     * This allows other repos, like dockstore-support, to use utility methods that require resource files from this package.
+     * @param resourceFileName
+     * @return
+     */
+    public static File getUniversalResourceFile(String resourceFileName) {
+        File tempResourceFile;
+        try (InputStream inputStream = Objects.requireNonNull(CommonTestUtilities.class.getClassLoader().getResourceAsStream(resourceFileName))) {
+            tempResourceFile = File.createTempFile(resourceFileName, null);
+            FileUtils.copyInputStreamToFile(inputStream, tempResourceFile);
+        } catch (IOException e) {
+            LOG.error("Could not get resource file {}", resourceFileName);
```

Comment:
In case `tempResourceFile` is created, but `copyInputStreamToFile` throws, we should delete `tempResourceFile` (if it exists) here.